cutouts/plot.py

```diff
@@ -86,6 +86,64 @@ def add_crosshair(
     )
     return
 
+
+#Adds circle around actual object  
+
+def add_circle(
+        ax: matplotlib.axes.Axes,
+        wcs: WCS,
+        obj_ra: float,
+        obj_dec: float,
+        rad: float = 4,
+        x_offset: int = 0,
+        y_offset: int = 0,
+        **kwargs
+    ):
+    """
+    Exercise of changing crosshairs into ring
+    Add a crosshair centered on RA and Dec to the given axes.
+
+    Parameter
+    ---------
+    ax : `~matplotlib.axes.Axes`
+        Matplotlib axes (usually a subplot) on which to add the crosshair.
+    wcs : `~astropy.wcs.wcs.WCS`
+        World Coordinate System (WCS) that maps pixels in an image to RA, Dec.
+    ra : float
+        Predicted RA in degrees.
+    dec : float
+        Predicted Dec in degrees.
+    gap : int
+        Distance from center in pixels to start drawing crosshair reticle bar.
+    length : int
+        Length in pixels of an individual bar reticle.
+    x_offset : int, optional
+        Offset in x-axis pixels from the sky-plane origin of the image (offsets might be non-zero
+        due to image centering, padding, and/or trimming).
+    y_offset : int, optional
+        Offset in y-axis pixels from the sky-plane origin of the image (offsets might be non-zero
+        due to image centering, padding, and/or trimming).
+    **kwargs
+        Keyword arguments to pass to ax.hlines and ax.vlines.
+    """
+    # Get pixel location of RA and Dec
+    y_center, x_center = wcs.world_to_array_index_values(obj_ra, obj_dec)
+    #y_center, x_center = wcs.world_to_pixel_values(ra, dec)
+    x_center = x_center + x_offset
+    y_center = y_center + y_offset
+
+    #Defining center and radius for the plotting circle function 
+    
+    center= (x_center, y_center)
+    
+    ax.add_patch(matplotlib.patches.Circle(center, rad, **kwargs))
+    
+    
+    
+    return
+
+
+
```

Comment:
Please tidy up this spacing. Also, `return` is not needed when the function doesn't actually return anything, so you can remove it.

```suggestion
    center = (x_center, y_center)
    ax.add_patch(matplotlib.patches.Circle(center, rad, **kwargs))
    
    
```